searchbird/config/test.scala

```diff
@@ -1,7 +1,7 @@
+import brigade.searchbird.config.SearchbirdServiceConfig
```

Comment:
seems weird that this scala code exists under searchbird/config, since we avoid doing this in our own codebase.